src/react/databrowser/objects/ObjectHead.jsx

```diff
@@ -1,14 +1,17 @@
-// @flow
+// @noflow
 import * as L from '../../ui-elements/ListLayout2';
 import React from 'react';
 
 type Props = {
-    bucketNameParam?: string,
+    bucketName?: string,
 };
-export default function ObjectHead({ bucketNameParam }: Props){
+export default function ObjectHead({ bucketName }: Props){
     return <L.Head>
         <L.HeadSlice>
-            {bucketNameParam}
+            <L.HeadContainer>
```

Comment:
Let's try to be consistant with BucketHead.jsx ```<L.Head> <HeaderContainer>  <L.HeadSlice> ... </L.HeadSlice> <L.HeadBody> ... </L.HeadBody> </HeaderContainer> </L.Head>```

Reply:
done